app/packs/src/apps/mydb/elements/details/reactions/schemeTab/ReactionDetailsScheme.js

```diff
@@ -692,6 +693,8 @@ export default class ReactionDetailsScheme extends Component {
         } else {
           sample.coefficient = updatedSample.coefficient;
         }
+        const rId = sample.belongTo ? sample.belongTo.id : null;
```

Comment:
I am following the old used convention. I would recommend changing old convention as well, since both are interconnected (what I think is out of scope)